### ISSUE TITLE
"Error: Request returned status code: 502" in getVSCodeVersions()

### ISSUE THREAD

**Issue by Foxhoundn** [redhat-developer/vscode-extension-tester#191]
This started happening on Sunday, never seen it before.

```
Extension 'api-handler.vsix' was successfully installed.
Error: Request returned status code: 502
Details: <html>
    <head>
        <title> Server Error </title>
    </head>
    <body>
        <font color =\"#aa0000\">
            <h2>Server Error.</h2>
        </font>
        There was an unexpected error in the request processing.
    </body>
</html>
at Request._callback (/home/test/testing/mP2EkAeFGFma0AxG2SXfFFeT/qorus-vscode/ui-test/node_modules/vscode-extension-tester/out/util/codeUtil.js:62:33)
    at Request.self.callback (/home/test/testing/mP2EkAeFGFma0AxG2SXfFFeT/qorus-vscode/ui-test/node_modules/request/request.js:185:22)
    at Request.emit (events.js:310:20)
    at Request.<anonymous> (/home/test/testing/mP2EkAeFGFma0AxG2SXfFFeT/qorus-vscode/ui-test/node_modules/request/request.js:1154:10)
    at Request.emit (events.js:310:20)
    at IncomingMessage.<anonymous> (/home/test/testing/mP2EkAeFGFma0AxG2SXfFFeT/qorus-vscode/ui-test/node_modules/request/request.js:1076:12)
    at Object.onceWrapper (events.js:416:28)
    at IncomingMessage.emit (events.js:322:22)
    at endReadableNT (_stream_readable.js:1187:12)
    at processTicksAndRejections (internal/process/task_queues.js:84:21)
```

FYI: For a few days now, we run the tests one by one instead of all in one command (for each test the VSCode is shut down and booted up again), so maybe it could be related (maybe there is a polling limit on the azure website), but it wasn't happening a few days ago.

**Comment by jrichter1** [comment]
That's a first. Given the response, it looks more like an internal server error, rather than API rate limit. So far I haven't run into this, neither has the CI. So far I can only ask to keep monitoring the problem, hopefully it will get solve on microsoft's side. 
The request goes to this URL: https://vscode-update.azurewebsites.net/api/releases/stable - if you wish to check manually.

**Comment by jrichter1** [comment]
Should be fixed in `3.2.4`, feel free to give it a try.
